### PR TITLE
feat: add footer

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

Screenshot 2022-07-28 at 11 49 25 AM

##### Refers/Fixes

https://github.com/gitcoinco/grants-round/pull/207#pullrequestreview-1052653462

### CODE REVIEW

File: packages/round-manager/src/assets/landing/banner.svg [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<svg width="704" height="800" viewBox="0 0 704 800" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg width="704" height="700" viewBox="0 0 704 800" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (shavinac) [inline packages/round-manager/src/assets/landing/banner.svg:1]: why was the height changed here?

File: packages/round-manager/src/features/common/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { ReactComponent as GitcoinCommunityLogo } from "../../assets/gitcoincommunity-logo.svg"
+
+
+const navigation = [
+  {
+    name: 'GitHub',
+    href: 'https://github.com/gitcoinco/grants-round',
+    icon: (props: any) => (
+      <svg width="21" height="20" viewBox="0 0 21 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (chibie) [inline packages/round-manager/src/features/common/Footer.tsx:9]: `props` isn't used here as in L22

**Review by chibie — CHANGES_REQUESTED** [review]

File: packages/round-manager/src/assets/landing/banner.svg (hunk 1 above)
Comment (thelostone-mc) [reply packages/round-manager/src/assets/landing/banner.svg:1]: The original image I had exported didn't have the footer while the newer one did! 
That's why it's been updated

File: packages/round-manager/src/features/common/Footer.tsx (hunk 2 above)
Comment (thelostone-mc) [reply packages/round-manager/src/features/common/Footer.tsx:9]: Oh good catch

Comment (thelostone-mc) [reply packages/round-manager/src/features/common/Footer.tsx:9]:
```suggestion
    icon: (props: any) => (
      <svg width="21" height="20" viewBox="0 0 21 20" fill="none" {...props} xmlns="http://www.w3.org/2000/svg">
```

**Review by chibie — APPROVED** [review]

**Pull request merged**